.github/workflows/ci-image.yml

```diff
@@ -8,4 +8,4 @@ jobs:
         uses: actions/checkout@v3.1.0
       - run: |
           echo ${{ secrets.GITHUB_TOKEN }} | docker login ghcr.io -u $GITHUB_ACTOR --password-stdin
-          docker buildx build --provenance=false --platform linux/amd64,linux/arm64 -t ghcr.io/${GITHUB_REPOSITORY}:ci-2 --push .github
+          docker buildx build --provenance=false --platform linux/amd64,linux/arm64 -t ghcr.io/${GITHUB_REPOSITORY}:ci-3 --push .github
```

Comment:
Why don't we use `latest`?

Reply:
We want to use labeled CI images so we can switch over in a pull request in the solang repo. Otherwise we have breakage when there are breaking changes like a new llvm version.